EG23.tex

```diff
@@ -268,23 +277,32 @@ \subsubsection{Case 1: Greedy Separations}
 the separation rate indirectly using the gradual fuel fabrication capacity
 growth. Full fabrication facilities stop requesting separated plutonium
 leading to eventual halting of spent fuel separations.
+As the separation output buffers are not limited, those separation facilities will
+reprocess all the used available fuel with a constant reprocessing rate.
 
+As the fuel fabrication facilities have a tuned deployement and capacities (for 10
+batchs of fuel), we are reporting for ``Fuel Resource in Storage'' is based on the
+inventories of the separation facilities plutonium output buffer.
+The reprocessed uranium is sent after the sepration to a dedicateed storage.
+
+Notable are subtle differences in what these two storage facilities data represent
+and their exact semantic meaning in the analysis spreadsheet.
 %% NOTE: if LWR separations output buffers are unlimited, it will never stop
 %% separating, right?
 
 \begin{table}[h!]
     \centering
     \begin{tabular}{lllll}
     \hline
-    Separation Properties	&	LWR			&	SFR 1/2	\\
+    Separation Properties     &  LWR        &  SFR 1/2    \\
     \hline
-    Throughput [tML/month]	&	83.3333		&	5000		\\
-    feed buffer [tML]		&	107.537		&	5000		\\
-    Pu output buffer [tML]	&	Unlimited	&	5000		\\
-    Pu separation efficiency&	0.99		&	0.99		\\
-    Recycled U buffer [tML]	&	Unlimited	&	Unlimited	\\
-    U separation efficiency	&	0.99		&	0.99		\\
-    Waste buffer [tML]		&	Unlimited	&	Unlimited	\\
+    Throughput [tHM/month]    &  83.3333    &  5000       \\
+    feed buffer [tHM]         &  107.537    &  5000       \\
+    Pu output buffer [tHM]    &  Unlimited  &  5000       \\
+    Pu separation efficiency  &  0.99       &  0.99       \\
+    Recycled U buffer [tHM]   &  Unlimited  &  Unlimited  \\
+    U separation efficiency   &  0.99       &  0.99       \\
+    Waste buffer [tHM]        &  Unlimited  &  Unlimited  \\
     \hline
```

Comment:
@gonuke I have rethink the reactor table, with 2 part : 
- the specification
- the implementation/information require by cyclus

I'll reformat all table that way...